### DISCUSSION TITLE
Tags index: Sorting pages by title instead of path

### DISCUSSION THREAD

**Discussion by normandy7** [squidfunk/mkdocs-material#5612] [Q&A] [answered]
Is it possible to configure the tags index so that the list of pages is sorted by something other than the file path?

Sorting by page title would make more sense from a reader perspective, as it looks rather arbitrary otherwise (unless the file paths alphabetically happen to match the titles, which is not the case for us).

**Comment by squidfunk** [comment]
That's currently not possible, but definitely something we can add. Pages should currently be listed in the order that they're defined in `mkdocs.yml` or by file path, if you do not specify a `nav` entry. You can open a change request if you like!

**Reply by normandy7** [reply]
Cheers, will do!

**Comment by squidfunk** [comment, marked as answer]
This is available in [Insiders](https://squidfunk.github.io/mkdocs-material/insiders/) via [`listings_sort_by`](https://squidfunk.github.io/mkdocs-material/plugins/tags/#config.listings_sort_by), e.g.:

``` yaml
plugins:
  - tags:
      listings_sort_by: !!python/name:material.plugins.tags.item_title
```

You can also provide a custom comparison function.

**Reply by normandy7** [reply]
I just tested Insiders `4.53.1` and for some reason the tags listing still comes out empty. So we're confined to `4.47.1` for now.

Thanks anyway!

**Reply by squidfunk** [reply]
This sounds like a bug or misconfiguration. If you can provide a minimal reproduction, we'll look into it immediately.
